addons/stock/models/stock_quant.py

```diff
@@ -92,7 +92,7 @@ def _domain_product_id(self):
         'Available Quantity',
         help="On hand quantity which hasn't been reserved on a transfer, in the default unit of measure of the product",
         compute='_compute_available_quantity')
-    in_date = fields.Datetime('Incoming Date', readonly=True)
+    in_date = fields.Datetime('Incoming Date', readonly=True, required=True, default=fields.Datetime.now())
```

Comment:
The default value will always be the same, i.e. `Datetime.now` value when launching the server.
It should be either `default=lambda self: fields.Datetime.now()` or `default=fields.Datetime.now` ;)

Reply:
Oups, my bad. You alright! Thx

Reply:
I am going to create a PR fix!

Reply:
odoo/odoo#63387